### PR TITLE
add hclog vet to check step

### PR DESCRIPTION
**Pull request by drewbailey**
run hclogvet to ensure that key/value log statement pairs are balanced

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -195,6 +196,9 @@ $(git-dir)/hooks/%: dev/hooks/%
 check: ## Lint the source code
 	@echo "==> Linting source code..."
 	@golangci-lint run -j 1
+	
+	@echo "==> Linting hclog statements"
```
Comment (shoenig) [inline GNUmakefile:200]: gotta add the `...` to fit in with the other echos :slightly_smiling_face:

**Comment by drewbailey** [comment]
if golanci-lint supports govet then yes, I'll take a quick peek and see if I can get it going quick

File: GNUmakefile [hunk 2]
```diff
@@ -183,6 +183,7 @@ lint-deps: ## Install linter dependencies
 	@echo "==> Updating linter dependencies..."
 	GO111MODULE=on cd tools && go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.24.0
 	GO111MODULE=on cd tools && go get github.com/client9/misspell/cmd/misspell@v0.3.4
+	GO111MODULE=on cd tools && go get github.com/hashicorp/go-hclog/hclogvet@master
```
[the remark this answers is not in the document]
Comment (drewbailey) [reply GNUmakefile:186]: missed this, yes will follow up

[the remark this answers is not in the document]
Comment (drewbailey) [reply GNUmakefile:186]: actually I'm not sure we can? its a submodule of go-hclog. I can pick a specific sha

```
GO111MODULE=on cd tools && go get github.com/hashicorp/go-hclog/hclogvet@v0.14.1
go get github.com/hashicorp/go-hclog/hclogvet@v0.14.1: github.com/hashicorp/go-hclog/hclogvet@v0.14.1: invalid version: unknown revision hclogvet/v0.14.1
make: *** [GNUmakefile:186: lint-deps] Error 1
```

**Pull request merged**